graphql-server/src/databases/database.resolver.ts

```diff
@@ -76,6 +82,26 @@ export class DatabaseResolver {
     return conn.currentDatabase();
   }
 
+  @Query(_returns => DatabaseConnection, { nullable: true })
+  async currentConnection(): Promise<DatabaseConnection | undefined> {
+    const config = this.conn.getWorkbenchConfig();
+    if (!config) return undefined;
+    const isDolt = this.conn.getIsDolt();
```

Comment:
Can you use `this.conn.isDolt`?

Reply:
`isDolt` is not accessible from `this.conn`

Reply:
Ah right. Was thinking `this.conn.connection()`